pkgs/tools/security/hash-identifier/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitLab, python3, python3Packages }:
```

Comment:
python3, or python3Packages, but not both

`python3.pkgs` -> `python3Packages`

Reply:
Okay good to know. The creddump package has both, and I basically copied it. I'll send in a PR to fix it as well.